src/components/NavigationBar/__snapshots__/NavigationBar.test.tsx.snap

```diff
@@ -18,7 +18,7 @@ exports[`NavigationBar > renders no navigation items and no user 1`] = `
         class="ml-5"
       >
         <button
-          aria-label="Valitse kieli"
+          aria-label="select-language"
```

Comment:
I could have mocked the translations but did not see the benefits as we are testing the NavigationBar component and mocking would not give actual benefit in this.